src/components/common/Detail.vue

```diff
@@ -54,10 +54,7 @@ export default class Detail extends Mixins(FilingTemplateMixin) {
 
   protected comment: string = null
 
-  get maxLength (): number {
-    // = (max size in db) - (default comment length) - (Carriage Return)
-    return (4096 - this.defaultCorrectionDetailComment.length - 1)
-  }
+  protected maxLength: number = 1000
```

Comment:
With the default correction detail (eg, "Correction for Incorporation Application filed on 2022-07-11."), the overall length will be over 1000.

I can't tell from the ticket what the requirement is.

Reply:
Yui commented on [12576](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/12576) and I assumed that it meant it would be 1000.

Reply:
I heard in grooming today that it's 1000 characters including the default correction detail, so you just need to update the original getter.